browser/css/device-mobile.css

```diff
@@ -517,6 +517,25 @@ nav:not(.spreadsheet-color-indicator) ~ #toolbar-wrapper #toolbar-up.w2ui-toolba
 #toolbar-down table.w2ui-button .w2ui-tb-image {
 	margin: 5px 9px !important;
 }
+#toolbar-down table.w2ui-button .textcolor {
+	margin: 5px 9px 0px !important;
+}
+#toolbar-down table.w2ui-button .backcolor {
+	margin: 5px 9px 0px !important;
+}
+#toolbar-down table.w2ui-button .backgroundcolor {
+	margin: 5px 9px 0px !important;
+}
```

Comment:
I have tested and as far as I can see we don't use `.backgroundcolor` on mobile #toolbar-down so this can be removed